api/capi/include/tensor_filter_single.h

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (C) 2019 Parichay kapoor <pk.kapoor@samsung.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ */
+/**
+ * @file	tensor_filter_single.h
+ * @date	28 Aug 2019
+ * @brief	Element to use general neural network framework individually without gstreamer pipeline
+ * @see	  http://github.com/nnsuite/nnstreamer
+ * @author	Parichay Kapoor <pk.kapoor@samsung.com>
+ * @author	MyungJoo Ham <myungjoo.ham@samsung.com>
+ * @bug	  No known bugs except for NYI items
+ */
+
+#ifndef __G_TENSOR_FILTER_SINGLE_H__
+#define __G_TENSOR_FILTER_SINGLE_H__
+
+#include <stdint.h>
+#include <glib-object.h>
+
+#include "nnstreamer_subplugin.h"
+#include "nnstreamer_plugin_api_filter.h"
```

Comment:
They are not in your local directory, try to use <nnstreamer/*.h> and drop nnsinc.

Reply:
updated.